android/build.gradle

```diff
@@ -7,6 +7,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.github.kezong:fat-aar:1.3.8'
```

Comment:
fat-aar may not work as you expected.
engine.jar bundle in aar won't work, it need to be dexed first.

I think we should just publish a pom only package as normal java project.